hbase-oss/src/test/java/org/apache/hadoop/hbase/oss/contract/TestHBOSSContractRootDirectory.java

```diff
@@ -34,6 +36,14 @@ protected Configuration createConfiguration() {
 
   @Override
   protected AbstractFSContract createContract(Configuration conf) {
-    return new HBOSSContract(conf);
+    HBOSSContract contract = new HBOSSContract(conf);
+    try {
+      TestUtils.getFileSystem(conf);
+    } catch (Exception e) {
+      e.printStackTrace();
+      fail("Exception configuring FS: " + e);
```

Comment:
better: throw and Assertion error with the inner cause. And given it's commonality, share this code across the tests

Reply:
Maybe make _TestUtils.getFileSystem(conf);_ throw _AssertionError_, since it's a basic test requirement? That would simplify this check for the subset of tests requiring it.

Reply:
Right, moved this logic to a new method in TestUtils that's then called on S3A specific contract tests.